app.R

```diff
@@ -268,37 +277,37 @@ server <- function(input, output, session) {
       leaflet.extras::addGeoJSONChoropleth(json_data,
-        valueProperty = "AREASQMI",
-        scale = c("white", "red"),
-        mode = "q",
-        steps = 4,
-        padding = c(0.2, 0),
-        labelProperty = "name",
-        popupProperty = propstoHTMLTable(
-          props = c("name", "description", "altitudeMode", "extrude"),
-          table.attrs = list(class = "table table-striped table-bordered"),
-          drop.na = TRUE
-        ),
-        color = "#43a858", weight = 1, fillOpacity = 0.7,
-        highlightOptions = highlightOptions(
-          weight = 2, color = "#9c4e57",
-          fillOpacity = 1, opacity = 1,
-          bringToFront = TRUE, sendToBack = TRUE
-        ),
-        pathOptions = pathOptions(
-          showMeasurements = TRUE,
-          measurementOptions =
-            measurePathOptions(imperial = TRUE)
-        ),
-        group = "District boundaries"
+                                           valueProperty = "AREASQMI",
+                                           scale = c("white", "red"),
+                                           mode = "q",
+                                           steps = 4,
+                                           padding = c(0.2, 0),
+                                           labelProperty = "name",
+                                           popupProperty = propstoHTMLTable(
+                                             props = c("name", "description", "altitudeMode", "extrude"),
+                                             table.attrs = list(class = "table table-striped table-bordered"),
+                                             drop.na = TRUE
+                                           ),
+                                           color = "#43a858", weight = 1, fillOpacity = 0.7,
+                                           highlightOptions = highlightOptions(
+                                             weight = 2, color = "#9c4e57",
+                                             fillOpacity = 1, opacity = 1,
+                                             bringToFront = TRUE, sendToBack = TRUE
+                                           ),
+                                           pathOptions = pathOptions(
+                                             showMeasurements = TRUE,
+                                             measurementOptions =
+                                               measurePathOptions(imperial = TRUE)
+                                           ),
+                                           group = "district_boundaries"
       ) %>%
-      hideGroup(group = "District boundaries") %>%
+      hideGroup(group = "district_boundaries") %>%
       # This is to add control layers on the map
       leaflet::addLayersControl(
         position = "bottomleft",
         baseGroups = c("Light"),
         overlayGroups =
-          c("District boundaries"),
+          c("district_boundaries"),
```

Comment:
@siddhant3030 any reason for this change. In UI button it was showing `district_boundaries` so I changed to `District boundaries` which lookes better UI wise

Reply:
See the logic above. If you change it here then you need to change the name of group which uses the same name.

Reply:
anyways I have updated it